jax/numpy/lax_numpy.py

```diff
@@ -4028,7 +4026,7 @@ def lcm(x1, x2):
   x1, x2 = _promote_dtypes(x1, x2)
   d = gcd(x1, x2)
   return where(d == 0, lax._const(d, 0),
-               lax.div(lax.abs(multiply(x1, x2)), d))
+               abs(multiply(x1, floor_divide(x2, d))))
```

Comment:
Note: switched order of operations here to prevent overflow for large inputs where possible.